server/controllers/routes/login.js

```diff
@@ -25,11 +24,11 @@ const login = (req, res, next) => {
         throw validationErr;
       }
       const { password: hashedPassword, parent_id: parentId, id } = rows[0];
-      return {
-        value: bcrypt.compare(password, hashedPassword),
+      return bcrypt.compare(password, hashedPassword).then((value) => ({
+        value,
         parentId,
         id,
-      };
+      }));
     })
```

Comment:
i think you have to return bcrypt.compare (as a Promise) then use this promise. return that to the previous way